django_tenants/tests/test_middleware.py

```diff
@@ -22,3 +33,25 @@ def test_invalid_hostname_should_return_404(self):
         response = self.client.get('/')
 
         self.assertIsInstance(response, HttpResponseNotFound)
+
+
+class ShowPublicNoTenantFound(FastTenantTestCase):
```

Comment:
Question: if I want to display something from the DB what's the current schema context in which this custom view operates?

Reply:
It operates in the public schema, 
<img width="810" alt="image" src="https://github.com/user-attachments/assets/afee7597-5ab1-4e5c-911b-ab037be975e7">

I'm using this settings in production currently :)